src/core_plugins/kibana/common/tutorials/heartbeat_instructions.js

```diff
@@ -0,0 +1,612 @@
+      }),
+      textPre: i18n.translate('kbn.common.tutorials.heartbeatCloudInstructions.config.debTextPre', {
+        defaultMessage: 'Modify {path} to set the connection information for Elastic Cloud:',
+        values: {
+          path: '`/etc/heartbeat/heartbeat.yml`',
+        },
+      }),
+      commands: [
+        'cloud.id: "{config.cloud.id}"',
+        'cloud.auth: "elastic:<password>"'
+      ],
+      textPost: i18n.translate('kbn.common.tutorials.heartbeatCloudInstructions.config.debTextPost', {
+        defaultMessage: 'Where {passwordTemplate} is the password of the `elastic` user.',
+        values: { passwordTemplate: '`<password>`' },
+      }),
+    },
+    RPM: {
+      title: i18n.translate('kbn.common.tutorials.heartbeatCloudInstructions.config.rpmTitle', {
+        defaultMessage: 'Edit the configuration',
+      }),
+      textPre: i18n.translate('kbn.common.tutorials.heartbeatCloudInstructions.config.rpmTextPre', {
+        defaultMessage: 'Modify {path} to set the connection information for Elastic Cloud:',
+        values: {
+          path: '`/etc/heartbeat/heartbeat.yml`',
+        },
+      }),
+      commands: [
+        'cloud.id: "{config.cloud.id}"',
+        'cloud.auth: "elastic:<password>"'
+      ],
+      textPost: i18n.translate('kbn.common.tutorials.heartbeatCloudInstructions.config.rpmTextPost', {
+        defaultMessage: 'Where {passwordTemplate} is the password of the `elastic` user.',
+        values: { passwordTemplate: '`<password>`' },
+      }),
+    },
+    WINDOWS: {
+      title: i18n.translate('kbn.common.tutorials.heartbeatCloudInstructions.config.windowsTitle', {
+        defaultMessage: 'Edit the configuration',
+      }),
+      textPre: i18n.translate('kbn.common.tutorials.heartbeatCloudInstructions.config.windowsTextPre', {
+        defaultMessage: 'Modify {path} to set the connection information for Elastic Cloud:',
+        values: {
+          path: '`C:\\Program Files\\Heartbeat\\heartbeat.yml`',
+        },
+      }),
+      commands: [
+        'cloud.id: "{config.cloud.id}"',
+        'cloud.auth: "elastic:<password>"'
+      ],
+      textPost: i18n.translate('kbn.common.tutorials.heartbeatCloudInstructions.config.windowsTextPost', {
+        defaultMessage: 'Where {passwordTemplate} is the password of the `elastic` user.',
+        values: { passwordTemplate: '`<password>`' },
+      }),
+    }
+  }
+});
+
+export function heartbeatEnableInstructions(moduleName) {
+  return {
+    OSX: {
+      title: i18n.translate('kbn.common.tutorials.heartbeatEnableInstructions.osxTitle', {
+        defaultMessage: 'Modify heartbeat.yml to configure the monitors',
+      }),
+      textPre: i18n.translate('kbn.common.tutorials.heartbeatEnableInstructions.osxTextPre', {
+        defaultMessage: 'Modify the settings in the `heartbeat.yml` file.',
+      }),
+      commands: [
+        '- type: http',
+        '  schedule: "@every 5s"',
+        '  urls: ["http://myhost:80"]'
```

Comment:
@dov0211 I wonder what our default here should be? `elastic.co`?

Reply:
Lets use the same structure as we have in our heartbeat.yml file
 urls: ["http://localhost:9200"]
 schedule: '@every 10s'

Reply:
SGTM.

For the cloud we discussed online to use `elastic.co` instead of `localhost:9200` as the local ES instance will not be available.